andela/mrm_api
# #164830375 Admin should be able to mark feedback responses as resolved

#### What does this PR do?
- Allow an Admin to mark feedback responses as resolved
#### Description of Task to be completed?
- Given I am an admin
- When feedback has been attended to 
- Then I should be able to mark it as resolved
#### How should this be manually tested?
- git pull and checkout branch ft-mark-feedback-responses-resolved-164830375
- run migrations to created the resolved column
- run application
- run mutation below
```
mutation{
  resolveRoomResponse(responseId:1){
    roomResponse{
      resolved
      id
      rate
      roomId
    }
  }
}

```
- Confirm in your local db if the resolved field for that response as changed to true.
#### Any background context you want to provide?
- Once an admin marks a response as resolved, he can unmark it by hitting the endpoint again.
#### What are the relevant pivotal tracker stories?
[#164830375](https://www.pivotaltracker.com/story/show/164830375)
#### Screenshots
##### Expected responses for that query 
<img width="867" alt="Screen Shot 2019-04-01 at 15 28 54" src="https://user-images.githubusercontent.com/39129473/55327531-fa21fa80-5492-11e9-9b2b-cb2099a287c1.png">


- On hitting the endpoint the second time
<img width="919" alt="Screen Shot 2019-04-01 at 15 29 01" src="https://user-images.githubusercontent.com/39129473/55327568-05752600-5493-11e9-9d01-fa3d30e1f02e.png">
